### ISSUE TITLE
Windows app now opening

### ISSUE THREAD

**Issue by Droz23-Real** [CorvusCodex/RouletteAi#1]
i dl'd the 4 items from the coffee website but after extraction Windows say "can not open for PC". idk how to fix this

**Comment by CorvusCodex** [comment]
Hello!
Thanks for support!
Use winrar. Extract first and then extract every recomended file in next steps. then run the app. The app supports at least windows 10.

**Comment by Droz23-Real** [comment]
The 225508_dist.z01 is missing on the zip version 225508. says the volume is missing.  there is the volumes for 225511, 225513, 225513 versions. The 225511_distz01 I can extract and use but idk what that version is.  . it bring up the console, runs the epochs and give a result. However it was that tensor autograph is not available. IDK if it is working correct. am i doing it correct?

**Comment by CorvusCodex** [comment]
Yeah you must pick recomended file when winrar ask you to, dont look at the name just recomended files when click locate. Thats becsuse Buymeacoffe change names but winrar knows what recomended file is. The tensor flow notice is normal for windows devices. Great you have made it!

**Comment by CorvusCodex** [comment]
Good luck! Take a look my at other apps, memberships or even support :) Greetings

**Comment by Droz23-Real** [comment]
![Screenshot 2023-11-16 161448](https://github.com/CorvusCodex/RouletteAi/assets/57382369/ddfc7114-3bda-4154-be01-dcb79138b4f3)
This the error for 225508dist.z01 How do i fix that?

**Comment by CorvusCodex** [comment]
click browse and click on first file that appear

**Comment by CorvusCodex** [comment]
and so on every prompt

**Comment by Droz23-Real** [comment]
![Screenshot 2023-11-16 161834](https://github.com/CorvusCodex/RouletteAi/assets/57382369/e59b7b6a-6332-4b14-8661-a662e3343ad7)

**Comment by Droz23-Real** [comment]
if I create a folder to extract it keeps says error. doesn't exist.

**Comment by CorvusCodex** [comment]
No, just extract one. When prompt browser apper click on the first file and so on.

**Comment by CorvusCodex** [comment]
So click browse, click on first file then open than ok, ans on every prompt the same. I have just tested it again.

**Comment by CorvusCodex** [comment]
And of course start with 225508_dist

**Comment by Droz23-Real** [comment]
I am idiot. Had to put the Zip file and the .z0"x"s  in the same folder first then extract 22508. Thanks. Haven't hit a number yet but it is fun to play with. Can you give me insight as to how it Predicts?

**Comment by CorvusCodex** [comment]
You made it! Well done! Based on learned patterns using historical data.

**Comment by Droz23-Real** [comment]
Last question. Does is run oldest outcome on the first line in the data.txt or the last?  first line or last line for every addition to the text file? i assume the newest data is the last line. I keep getting 34, 26, 35, 5, 12 at outcomes.
